Runtime/Core/AxisState.cs

```diff
@@ -210,11 +210,7 @@ public bool Update(float deltaTime)
                 return false;
             m_LastUpdateFrame = Time.frameCount;
 
-            // Cheating: we want the render frame time, not the fixed frame time
```

Comment:
you need to leave this comment, or something like it. What's going on isn't obvious because the caller is passing in deltaTime